dashboard/app/models/section.rb

```diff
@@ -40,6 +40,11 @@ class Section < ActiveRecord::Base
   LOGIN_TYPE_PICTURE = 'picture'
   LOGIN_TYPE_WORD = 'word'
 
+  TYPES = [
+    TYPE_PD_WORKSHOP = 'pd_workshop'
```

Comment:
Still, each constant is defined separately and is not referenced as a member of the container array. This pattern is a shorthand for:

```ruby
TYPE_PD_WORKSHOP = 'pd_workshop'
TYPES = [TYPE_PD_WORKSHOP]
```

I'm declaring `TYPES` as the list of all options for the validation, similar to [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/workshop.rb#L27-L71).

Unfortunately we can't refer to it as something like `TYPES.PD_WORKSHOP`. If we wanted that, we could use a hash of symbol keys instead of constants? Would this be preferable?

```ruby
TYPES = {
  pd_workshop: 'pd_workshop'
}

# referenced by
TYPES[:pd_workshop]
```